scripts/build/build.sh

```diff
@@ -24,10 +38,26 @@ source /etc/profile.d/rvm.sh
 
 echo "current dir: $(pwd)"
 
+if [ -d "dist" ]; then
+  rm -rf dist
+fi
+
 if [ "$CIRCLE_TAG" != "" ]; then
+  echo "Building frontend from tag $CIRCLE_TAG"
```

Comment:
As previous `then` part, `-buildNumber` not needed with `-includeBuildNumber=false